applications/test_treeAMR/CMakeLists.txt

```diff
@@ -5,18 +5,15 @@ add_executable(test_treeAMR "test_treeAMR.cpp")
 find_package(MPI REQUIRED)
 find_package(ZLIB REQUIRED)
 
-set_target_properties(test_treeAMR PROPERTIES CXX_STANDARD 17)
+set_target_properties(test_treeAMR PROPERTIES CXX_STANDARD 20)
```

Comment:
Isn't that set project wide in the via the main CMakeLists.txt? See https://cmake.org/cmake/help/latest/variable/CMAKE_CXX_STANDARD.html#variable:CMAKE_CXX_STANDARD

We set this variable so cmake will add `CXX_STANDARD` as a property to every target created within this project. We would only need to set something here if we want to enforce a different standard for a particular target.